configure.ac

```diff
@@ -304,6 +304,15 @@ changequote([,])dnl
 
 AC_CHECK_FUNCS(fdwalk)
 
+# Storaged
+
+AC_ARG_WITH(storaged-iscsi-sessions,
```

Comment:
I would suggest marking this as HACK: with the appropriate https://github.com/storaged-project/storaged/pull/68 link.